.gitignore

```diff
@@ -9,6 +9,7 @@ META.*
 MYMETA.*
 Makefile.PL
 *.gz
+!t/data/engine/**/reference-output/**/*.gz
```

Comment:
I generate output and gzip it before comparison, too. But I see you already took care of the timestamps in 1cacfed935e6947d81380b7d92b48b832513390e